app/containers/message/utils.ts

```diff
@@ -2,7 +2,7 @@ import { TMessageModel } from '../../definitions/IMessage';
 import I18n from '../../i18n';
 import { DISCUSSION } from './constants';
 
-export const formatMessageCount = (count?: number, type?: string): string => {
+export const formatMessageCount = (count?: number, type?: string) => {
```

Comment:
I'm sure this is a better approach
```suggestion
export const formatMessageCount = (count?: number, type?: string): string | null => {
```